index.js

```diff
@@ -104,8 +101,19 @@ module.exports = class Filter extends Plugin {
     return !!result;
   }
 
-  constructor(inputTree, options) {
-    super([inputTree], {
+  get fsMerger() {
```

Comment:
as implemented `fsMerger` is still  part of the public API for subclasses, this seems sorta strange right?